pkg/backtest/exchange.go

```diff
@@ -55,7 +55,7 @@ var ErrEmptyOrderType = errors.New("order type can not be empty string")
 type Exchange struct {
 	sourceName     types.ExchangeName
 	publicExchange types.Exchange
-	srv            *service.BacktestService
+	srv            service.IBacktestService
```

Comment:
we usually don't use "I" as the prefix for interface types.

It could be BacktestServiceProvider or Backtestable